wallet/manager.go

```diff
@@ -1,25 +1,56 @@
 package wallet
 
 import (
+	"os"
+	"path/filepath"
+
 	"github.com/pactus-project/pactus/crypto"
 	"github.com/pactus-project/pactus/crypto/bls"
 	"github.com/pactus-project/pactus/genesis"
 	"github.com/pactus-project/pactus/types/tx"
+	"github.com/pactus-project/pactus/util"
 	"github.com/pactus-project/pactus/wallet/vault"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 type Manager struct {
-	wallets   map[string]*Wallet
-	chainType genesis.ChainType
+	wallets         map[string]*Wallet
+	chainType       genesis.ChainType
+	walletDirectory string
 }
 
-func NewWalletManager(chainType genesis.ChainType) *Manager {
+func NewWalletManager(chainType genesis.ChainType, walletDir string) *Manager {
 	return &Manager{
-		wallets:   make(map[string]*Wallet),
-		chainType: chainType,
+		wallets:         make(map[string]*Wallet),
+		chainType:       chainType,
+		walletDirectory: walletDir,
+	}
+}
+
+func (w *Manager) walletExists(walletName string) bool {
+	walletPath := util.MakeAbs(filepath.Join(w.walletDirectory, walletName))
```

Comment:
We can avoid repeating this line by defining a function like:
`func getWalletPath(walletName string) string { ... }`